src-tauri/src/plugins/config.rs

```diff
@@ -136,6 +134,15 @@ pub fn set_config(config_manager: State<ConfigManager>, config: Config) {
     config_manager.update(config);
 }
 
+/**
+ * Get the app configuration/storage directory
+ */
+// TODO: Replace with PathResolver::app_config_dir().
+fn get_app_storage_dir() -> PathBuf {
+    let path = dirs::home_dir().expect("Get home dir");
```

Comment:
So, how about using `dirs::config_dir();`? + `museeks` as folder.

Reply:
`directories` would be more appropriate than the the low-level `dirs` crate.

But since this is supposed to be a temporary hack anything that respects the established standards on each platform is better than polluting the home directory.